docs/conf.py

```diff
@@ -140,7 +140,8 @@ def get_xsversion(self):
 ]
 # Imported from sphinx_astropy so we don't have to maintain the list
 # of servers
-# intersphinx_mapping = {'python': ('https://docs.python.org/3', None)}
+# but adding to the astropy list.
+intersphinx_mapping['bokeh'] = ('https://docs.bokeh.org/en/latest/', None)
 
```

Comment:
So we don't need the "python" mappnig rule then I guess?

Reply:
On top of the file, we do `from sphinx_astropy.conf.v1 import intersphinx_mapping`. We don't need the Python rule, because astropy uses that, too, and thus it's already in `sphinx_astropy.conf.v1.intersphinx_mapping` together with matplotlib, numpy, ans astropy itself. However, astropy does not use bokeh, and thus bokeh is not part of what we import, we have to add it to the dictionary ourselves.